package.json

```diff
@@ -1,28 +1,75 @@
 {
   "name": "kup-kudosapp-backend",
-  "version": "0.0.0",
-  "private": true,
+  "version": "1.0.0",
+  "description": "Kudos App backend",
+  "author": "Appnroll",
+  "license": "MIT",
   "scripts": {
-    "start": "node ./bin/www"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/Appnroll/kup-kudosapp-backend.git"
+    "postinstall": "npm run build && touch ./dist/production.env",
+    "build": "tsc -p tsconfig.build.json",
+    "format": "prettier --write \"src/**/*.ts\"",
+    "start": "cross-env NODE_ENV=development ts-node -r tsconfig-paths/register src/main.ts",
+    "start:dev": "nodemon",
+    "start:debug": "nodemon --config nodemon-debug.json",
+    "start:prod": "node dist/main.js",
+    "lint": "tslint -p tsconfig.json -c tslint.json",
+    "test": "jest",
+    "test:watch": "jest --watch",
+    "test:cov": "jest --coverage",
+    "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
+    "test:e2e": "jest --config ./test/jest-e2e.json",
+    "test:e2e:watch": "cross-env jest --watch --config ./test/jest-e2e.json --runInBand",
+    "migration:generate": "ts-node ./node_modules/typeorm/cli.js migration:generate -n",
+    "migration:run": "ts-node ./node_modules/typeorm/cli.js migration:run",
+    "migration:revert": "ts-node ./node_modules/typeorm/cli.js migration:revert"
   },
   "dependencies": {
-    "cookie-parser": "~1.4.3",
-    "debug": "~2.6.9",
+    "@nestjs/common": "^5.4.0",
+    "@nestjs/core": "^5.4.0",
+    "@nestjs/swagger": "^2.5.1",
+    "@nestjs/typeorm": "^5.2.2",
+    "@types/joi": "^14.0.1",
+    "@types/lodash": "^4.14.119",
     "dotenv": "^6.2.0",
-    "express": "~4.16.0",
-    "http-errors": "~1.6.2",
-    "jade": "~1.11.0",
+    "joi": "^14.3.1",
     "lodash": "^4.17.11",
-    "morgan": "~1.9.0",
+    "mysql": "^2.16.0",
```

Comment:
Nie używasz czasem Postgresa? 🤔

Reply:
Korzystałem z postgres'a i myslq'a. Z faktu, że hostujemy na heroku a tam raczej postgres (chociaz mysql też może być to) to racja zostało posgres. Mysql może wylecieć.